evap/static/scss/_adjustments.scss

```diff
@@ -140,6 +140,14 @@ $alert-colors: (
     white-space: normal;
 }
 
+.user-multi-select {
+    padding-right: 36px !important;
+}
+
+.user-multi-select li {
+    width: 100%;
+}
+
```

Comment:
since you're at it: please set a lighter background color like this:
```
.user-multi-select.select2-selection--multiple .select2-selection__choice {
    background-color: $lighter-gray;
}
```